x-pack/plugins/observability_solution/inventory/server/routes/entities/get_latest_entities.ts

```diff
@@ -25,23 +27,28 @@ export async function getLatestEntities({
 }: {
   inventoryEsClient: ObservabilityElasticsearchClient;
   sortDirection: 'asc' | 'desc';
-  sortField: string;
+  sortField: EntityColumnIds;
   entityTypes?: EntityType[];
   kuery?: string;
 }) {
-  const latestEntitiesEsqlResponse = await inventoryEsClient.esql('get_latest_entities', {
+  // alertsCount doesn't exist in entities index. Ignore it and sort by entity.lastSeenTimestamp by default.
+  const entitiesSortField = sortField === 'alertsCount' ? ENTITY_LAST_SEEN : sortField;
+
+  const request = {
     query: `FROM ${ENTITIES_LATEST_ALIAS}
      | ${getEntityTypesWhereClause(entityTypes)}
      | ${getEntityDefinitionIdWhereClause()}
-     | SORT ${sortField} ${sortDirection}
+     | SORT ${entitiesSortField} ${sortDirection}
```

Comment:
We've been talking about this, and maybe you could refactor it to use [ESQL Params](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-rest.html#esql-rest-params) now that you're changing it anyway.

Up to you ofc.

Reply:
Now both entitiesSortField and entityTypes are typed in the router, the user won't be able to pass any custom values, so I this we're good for now.